package-lock.json

```
@@ -1,12 +1,20 @@
 {
   "name": "cuttle",
-  "version": "4.3.21",
+<<<<<<< Updated upstream
+  "version": "4.3.29",
+=======
+  "version": "4.3.28",
+>>>>>>> Stashed changes
```

This is a merge conflict that needs resolving
```suggestion
  "version": "4.3.29",
```